rebasehelper/utils.py

```diff
@@ -323,7 +322,26 @@ def close(self):
         session = requests.Session()
         session.mount('ftp://', FTPAdapter())
 
-        r = session.get(url, stream=True)
+        try:
+            return session.get(url, **kwargs)
+        except requests.exceptions.RequestException as e:
+            logger.debug('%s: %s', type(e).__name__, six.text_type(e))
+            return None
+
+    @staticmethod
+    def download_file(url, destination_path, blocksize=8192):
+        """
+        Method for downloading file from HTTP, HTTPS and FTP URL.
+
+        :param url: URL from which to download the file
+        :param destination_path: path where to store downloaded file
+        :param blocksize: size in Bytes of blocks used for downloading the file and reporting progress
+        :return: None
+        """
+        r = DownloadHelper.request(url, stream=True)
+        if r is None:
+            raise DownloadError
```

Comment:
This means rebase-helper will just stop without any message printed on stdout. That's not very good user experience ;-)

Reply:
Oh, I totally forgot about this part. What would be an ideal message in this case? Maybe something like: `"Unable to connect to %s", %url`?

Reply:
I don't think printing URL again is necessary (as it has been printed already few lines above). Something like `Unexpected error occured during download, see ...log`. But perhaps the original error message should be also printed to stdout, not just to debug log.